modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/custom/form/components/PartialResults.scss

```diff
@@ -4,19 +4,24 @@
 	&--background {
 		background: $light;
 		min-height: 100%;
+
+		.ddm-form-builder-app {
+			margin-top: 0;
+			padding-top: 28px;
+		}
```

Comment:
I think this class should be placed into general Forms' style instead of the component's style file